### ISSUE TITLE
[1.18.2] Crash on start

### ISSUE THREAD

**Issue by Konstantinell** [the9grounds/AE-Additions#82]
I have a client  crash when this mod and mod [AE2-Additions](https://www.curseforge.com/minecraft/mc-mods/ae2-additions) are install together. Separate both mods work good.
Applied Energistics 2 version 11.1.4
AE Additions - ExtraCells2 Fork version 3.1.1
AE2Additions version 0.1.2
Forge version 40.1.54

[crash-2022-07-16_22.05.37-client.txt](https://github.com/the9grounds/AE-Additions/files/9126443/crash-2022-07-16_22.05.37-client.txt)

[latest.log](https://github.com/the9grounds/AE-Additions/files/9126442/latest.log)

**Comment by Beachman4** [comment]
Yes both of our mods have the same mod id and unfortunately that means they are incompatible with each other.

**Comment by SerStarStory** [comment]
You are right. Unfortunately, You or I "stole" the modid :0

**Comment by Beachman4** [comment]
> You are right. Unfortunately, You or I "stole" the modid :0

Well I don't think anyone "stole" anything. I think perhaps you may have been unaware of my mod when porting yours to 1.18.

Similar to me, I was unaware of MEGA when making my mod for 1.18.2, which adds bigger storage cells and crafting storage blocks. I only learned about it a few weeks later and made the decision to prefer MEGAs items in my recipes and hide my bigger cells when both AE Additions and MEGA are loaded.
